### ISSUE TITLE
Develop documentation for LandPKS co-design partners

### ISSUE THREAD

**Issue by CourtneyLee333** [techmatters/terraso-product#496]

**Comment by CourtneyLee333** [comment]
Waiting for response from Terraso product team email.

**Comment by CourtneyLee333** [comment]
@ltseng I don't know if we'll get much input from the team, so maybe you and I should take on this conversation with Laura next month.

**Comment by ltseng** [comment]
Makes sense. I think I was telling Amaya that waiting until after SRM when we have a bunch of contacts to put into Hubspot would be best for the timeline there.

**Comment by CourtneyLee333** [comment]
Well, there are thousands to add already if we move the contacts from MailChimp to Hubspot. I'll work on a list of things to discuss with Laura.
